packages/rum-recorder/src/domain/rrweb/record.ts

```diff
@@ -33,7 +33,7 @@ export function record(options: RecordOptions = {}): RecordAPI {
 
   const takeFullSnapshot = (isCheckout = false) => {
     const wasFrozen = mutationController.isFrozen()
-    mutationController.freeze() // don't allow any mirror modifications during snapshotting
+    mutationController.freeze() // don't allow any node to be serialized or mutation to be emitted during sharpshooting
```

Comment:
```suggestion
    mutationController.freeze() // don't allow any node to be serialized or mutation to be emitted during snapshooting
```

Reply:
Damn autocorrect :)

Reply:
I'd love to have a sharpshooting concept though 😄